class.jetpack-connection-banner.php

```diff
@@ -308,26 +321,21 @@ public static function render_connect_prompt_full_screen() {
 					<?php jetpack_render_tos_blurb(); ?>
                 </p>
 
-				<?php if ( \Automattic\Jetpack\Constants::is_true( 'JETPACK_SHOULD_USE_CONNECTION_IFRAME' ) ) : ?>
-					<div class="jp-connect-full__button-container">
-					</div>
-				<?php else: ?>
-					<p class="jp-connect-full__button-container">
-						<a href="<?php echo esc_url( Jetpack::init()->build_connect_url( true, false, $bottom_connect_url_from ) ); ?>"
-						   class="dops-button is-primary">
-							<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
+                <p class="jp-connect-full__button-container">
+					<a href="<?php echo esc_url( Jetpack::init()->build_connect_url( true, false, $bottom_connect_url_from ) ); ?>"
+					   class="dops-button is-primary jp-connect-button">
+						<?php esc_html_e( 'Set up Jetpack', 'jetpack' ); ?>
+					</a>
+				</p>
+
+				<?php if ( 'plugins' === $current_screen->base ) : ?>
+					<p class="jp-connect-full__dismiss-paragraph">
+						<a>
```

Comment:
Is this link functional? If not, should we add a `TODO` or something?

Reply:
I think it is, this was here already. we are bringing it back from: https://github.com/Automattic/jetpack/pull/13262